.gitignore

```diff
@@ -1,3 +1,4 @@
 # Local node installation
 node_modules/
 .docker-build
+packaged
```

Comment:
Still not happy about this name. The command is called "build-json". It's not building a "package". That word is too much "npm" to me in software language.

Reply:
I'm happy to change this. I had thought you would in your PR :)